.storybook/main.js

```diff
@@ -15,4 +18,23 @@ module.exports = {
       },
     },
   ],
+  webpackFinal: async (config) => {
+    config.module.rules.push({
+      test: /\/Icon\.js$/,
+      use: path.resolve(__dirname, "./icon-component-loader.js"),
+    });
+    const SVGSpritesPlugin = new SVGSpritemapPlugin(
+      path.resolve(__dirname, "../src/icons/**/*.svg"),
+      {
+        sprite: {
+          prefix: false,
+          generate: {
+            symbol: true,
+          },
+        },
+      },
+    );
+    config.plugins.push(SVGSpritesPlugin);
+    return config;
+  },
```

Comment:
This in `next` was a separate webpack config, lmk if should be that way

Reply:
Also not sure if it's working but storybook seems to run, build, and publish to Chromatic fine

Reply:
if it works it works! 👍